--- misc.py
+++ bundler/jhbuild/utils/misc.py
@@ -36,7 +36,11 @@ def inpath(filename, path):
 def try_import_module(module_name):
     """Like importlib.import_module() but doesn't raise if the module doesn't exist"""

-    if pkgutil.get_loader(module_name) is None:
+    try:
+        import importlib.util
+        if importlib.util.find_spec(module_name) is None:
+            return
+    except (ImportError, ModuleNotFoundError, ValueError):
         return
     return importlib.import_module(module_name)

